pkg/virt-launcher/notify-client/notify_test.go

```diff
@@ -104,7 +104,9 @@ var _ = Describe("Notify", func() {
 				mockDomain.EXPECT().GetXMLDesc(gomock.Eq(libvirt.DomainXMLFlags(0))).Return(string(x), nil)
 				mockDomain.EXPECT().GetMetadata(libvirt.DOMAIN_METADATA_ELEMENT, "http://kubevirt.io", libvirt.DOMAIN_AFFECT_CONFIG).Return(`<kubevirt></kubevirt>`, nil)
 
-				eventCallback(mockCon, util.NewDomainFromName("test", "1234"), libvirtEvent{Event: &libvirt.DomainEventLifecycle{Event: event}}, client, deleteNotificationSent, nil, nil)
+				domainCache := util.NewDomainFromName("test", "1234")
```

Comment:
What is a "domainCache"? Where this name came from?

Reply:
as the client does
https://github.com/kubevirt/kubevirt/blob/2d22cb6a9706e0ef0e1a9cc7b0b7fd1d57480730/pkg/virt-launcher/notify-client/client.go#L320

but in some cases it might be null, so i cant just use it in the last parameter.

Reply:
I just see it returns a `domain`, I do not understand what makes it a `cache`.